src/css/window_tool.css

```
@@ -119,45 +124,51 @@
     content: attr(data-label);
 }
 
-/* 対称・回転描画（曼荼羅/雪結晶） */
-#axp_tool_div_symmetry {
+/* 書出し・共有 / 参照画像 disclosure */
+.axp_tool_disclosure {
+    --axp-interop-compact-h: 22px;
     display: flex;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)